include/TPP/Dialect/Tpp/TppOps.td

```diff
@@ -50,21 +65,32 @@ def BroadcastableShape : NativeOpTrait<"tpp::BroadcastableShape">;
 // dimension is one.
 def UnitStrideInnerLoop : NativeOpTrait<"tpp::UnitStrideInnerLoop">;
 
+// Tpp operation trait - verify tpp operand segment size.
+def UnaryOp : NativeOpTrait<"tpp::UnaryOp">;
+def BinaryOp : NativeOpTrait<"tpp::BinaryOp">;
```

Comment:
Add temporary hacked check for ternary. We'll clean it up once matmul syntax is updated.